chowpronto_backend/controllers/menuitems-controller.js

```diff
@@ -0,0 +1,15 @@
+const MenuItem = require("../models/MenuItem");
+
+const getMenuItems = async (req, res) => {
+  try {
+    let allMenuItems = await MenuItem.find({ inStock: true }).populate(
+      "byVendor",
+      "name"
+    );
+    res.send(allMenuItems);
+  } catch (err) {
+    console.error(err.message);
+    res.status(500).send("Server Error");
```

Comment:
This is a good start. 

In the future, we'll need to discuss the type of error messages that we send back to the front end.